packages/trpc-server/src/routers/playlist-router.ts

```diff
@@ -9,5 +18,37 @@ export const playlistRouter = router({
       throw new TRPCError({ code: 'NOT_FOUND' })
     }
     return row
-  })
+  }),
+
+  containTrackId: publicProcedure
```

Comment:
This is so cool

Reply:
yeah it's super nice! Very excited to see this become the new pattern